presto-main/src/test/java/io/prestosql/server/ui/TestWebUi.java

```diff
@@ -438,6 +459,45 @@ public void testJwtAuthenticator()
         }
     }
 
+    @Test
+    public void testJwtWithJwkAuthenticator()
+            throws Exception
+    {
+        TestingHttpServer jwkServer = createTestingJwkServer();
```

Comment:
It would be nice if that would be closeable.

Reply:
Yes it would be nice